interface/patient_file/encounter/delete_form.php

```diff
@@ -82,7 +84,13 @@
 ?>
 <input type="hidden" id="confirm" name="confirm" value="1"/>
 <p>
-<?php echo xlt('You are about to delete the following form from this encounter') . ': ' . text(xl_form_title($_GET['formname'])); ?>
+<?php
+
+$formdir=$_REQUEST["formname"];
```

Comment:
Rec using GET. POST will not make it this far into the script and would create a potential csrf vulnerability.

Reply:
FIXED  - $_REQUEST TO $_GET
thnaks.
Dror G.